target_gsheet.py

```diff
@@ -8,6 +8,10 @@
 import json
 import logging
 import collections
+if sys.version_info.major == 3 and sys.version_info.minor >= 10:
```

Comment:
This is most commonly done with a tuple comparison

```suggestion
if sys.version_info >= (3, 10):
```

Reply:
Thanks for the suggestion - I made the change and also verified that `version_info` is compatible for several previous Python versions (which would have been an ironic omission had that not been the case!).